app/bin/service/search.dart

```diff
@@ -94,27 +94,33 @@ Future _main(FrontendEntryMessage message) async {
     _updateDartSdkIndex().whenComplete(() {});
 
     final BatchIndexUpdater batchIndexUpdater = BatchIndexUpdater();
-    await batchIndexUpdater.initSnapshot();
-
-    final scheduler = TaskScheduler(
-      batchIndexUpdater,
-      [
-        ManualTriggerTaskSource(taskReceivePort.cast<Task>()),
-        IndexUpdateTaskSource(db.dbService, batchIndexUpdater),
-        DatastoreHeadTaskSource(
-          db.dbService,
-          TaskSourceModel.scorecard,
-          sleep: const Duration(minutes: 10),
-          skipHistory: true,
-        ),
-        batchIndexUpdater.periodicUpdateTaskSource,
-      ],
+    // Don't block on init, we need to serve liveliness and readiness checks.
+    batchIndexUpdater.init().then(
```

Comment:
How about we do:
```dart
scheduleMicrotask(() async {
  await batchIndexUpdater.init();

  final scheduler = ...
  ....
});
```

Just to avoid implicit micro tasks with `.then`...